Corona-Warn-App/src/main/res/values/dimens.xml

```diff
@@ -148,4 +148,7 @@
     <dimen name="contact_diary_nested_recyclerview_margin">60dp</dimen>
     <dimen name="statistics_card_height">328dp</dimen>
     <dimen name="statistics_card_width">300dp</dimen>
+
+    <!-- Swipe to delete icons left or right margin from the edge-->
+    <dimen name="swipe_icon_margin">33dp</dimen>
```

Comment:
Subjektive, this doesn't seem like something that will be re-used and could be hardcoded?

Reply:
It just easier to keep it here